packages/integration-tests/helpers/testFaucet.js

```diff
@@ -48,27 +48,27 @@ const faucet = {
       this.privateKey = faucetAccount.privateKey
     }
 
-    await this.initEthBalance(web3, web3.utils.toWei(config.minAmountEth || '3', 'ether'))
+    await this.initEthBalance(web3, web3.utils.toWei(config.minAmountEth || '1', 'ether'))
```

Comment:
```suggestion
    await this.initEthBalance(web3, web3.utils.toWei(config.minAmountEth || '3', 'ether'))
```
Revert to 3 ETH - necessary to run the entire test suite